### PR TITLE
Adding networkId field in NS documentation

### PR DESCRIPTION
**Pull request by mcompastie**

### CODE REVIEW

**Review by nebilbenmabrouk — APPROVED** [review]
LGTM

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 1]
```diff
@@ -462,8 +462,10 @@ Properties needed for the deployment of instance are:
    -   *image:* Defines which image will be used to create the OpenStack instance. The value to provide is the unique image Id.
    
    -   *flavor:* Defines the size of the instance. The value to provide is the flavor name or the flavor Id. For more information see https://docs.openstack.org/openstack-ops/content/flavors.html[OpenStack flavors].
+
+   -   *networkId:*  Defines the identifier of the network to be attached to the instance. This value can be blanked to leave the selection to OpenStack.
```
Comment (mklkun) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:466]: can be blanked => can be blank

**Review by mklkun — APPROVED** [review]
One single comment.

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc (hunk 1 above)
Comment (mboussaa) [reply src/docs/admin/references/NodeSourceInfrastructureReference.adoc:466]: multiple network Ids could be provided? Why not mention that in the doc

Comment (mcompastie) [reply src/docs/admin/references/NodeSourceInfrastructureReference.adoc:466]: @mboussaa : Noop, that's not the case here.

Comment (mboussaa) [reply src/docs/admin/references/NodeSourceInfrastructureReference.adoc:466]: ok

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 2]
```diff
@@ -462,8 +462,10 @@ Properties needed for the deployment of instance are:
    -   *image:* Defines which image will be used to create the OpenStack instance. The value to provide is the unique image Id.
    
    -   *flavor:* Defines the size of the instance. The value to provide is the flavor name or the flavor Id. For more information see https://docs.openstack.org/openstack-ops/content/flavors.html[OpenStack flavors].
+
+   -   *networkId:*  Defines the identifier of the network to be attached to the instance. This value can be blank to leave the selection to OpenStack.
```
Comment (mboussaa) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:466]: You can improve like this (but it is up to you): This value can be blank to leave the selection to OpenStack => The network Id is selected/defined by OpenStack when it is not provided

**Pull request merged**